.github/workflows/coveo-arnparse.yml

```
@@ -0,0 +1,62 @@
+name: coveo-arnparse
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'coveo-arnparse/**'
+      - '.github/workflows/**'
+
+  pull_request:
+    types: [opened, synchronize, reopened]
+    paths:
+      - 'coveo-arnparse/**'
+      - '.github/workflows/**'
+
+jobs:
+  stew:
+    name: stew ci
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.8, 3.9]
+        os: [ubuntu-20.04, windows-2019, macos-10.15]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Setup python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Run stew ci
+        uses: ./.github/workflows/actions/stew-ci
+        with:
+          project-name: ${{ github.workflow }}
+
+  publish:
+    name: Publish to pypi.org
+    runs-on: ubuntu-20.04
+    needs: pyprojectci
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Setup python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+
+      - name: Publish to pypi
+        uses: ./.github/workflows/actions/publish-to-pypi
+        with:
+          project-name: ${{ github.workflow }}
+          pypi-token: ${{ secrets.PYPI_TOKEN_COVEO_ARNPARSE }}
+          pre-release: ${{ github.ref != 'refs/heads/main' }}
+          dry-run: true
```

```suggestion
          dry-run: ${{ github.ref != 'refs/heads/main' && github.event.inputs.publish != 'true' }}
```

All other projects use this in their workflow. Seems like it'd help when it comes time to release

---

🤦🏻‍♂️